pkgs/comma/default.nix

```diff
@@ -24,7 +24,7 @@ naersk-lib.buildPackage {
   overrideMain = _: {
     postInstall = ''
       wrapProgram $out/bin/comma \
-        --prefix PATH : ${
+        --suffix PATH : ${
```

Comment:
Why introduce impurities here? The original `--prefix` seems like the more stable choice